api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -321,3 +336,59 @@ export const insertSurveySubmissionMessageSQL = (
 
   return sqlStatement;
 };
+
+/**
+ * SQL query to get the list of errors for an occurrence submission.
```

Comment:
this seems incorrect as the comment here
it doesnt get errors it gets messages

Reply:
nice catch, thanks!